src/test/java/edu/hillel/lesson24/StorageFactoryTest.java

```diff
@@ -0,0 +1,32 @@
+package edu.hillel.lesson24;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StorageFactoryTest {
+
+    private StorageFactory storageFactory;
+
+    @Before
+    public void setUp() throws Exception {
+        storageFactory = new StorageFactory();
+    }
+
+    @Test
+    public void createStorage() {
+        Storage storage = storageFactory.createStorage(StorageType.IN_MEMORY);
```

Comment:
What about other types of storages?

Reply:
I test method createStorage - if I set storage type=A - I get storage type of A class.
Yes - I can add tests for all types - but I don't think that it's needed.
Testing of correct work of all storage types exists in each classes.
Test of method storageFactory.createStorage - I've done - one test for correct and and one for exception.

Reply:
What is somebody changes internal implementation inside createStorage? For example, change creating a DB storage?

Reply:
I understand you, but count of types can be very big and count of differences can also be very big.
For example - calculator and operation sum (a+b)
We make test that check than 1+2=3
But in method - smebody can change it implementation and do so
if (a==10){
return a*15-b+8;
}
else{
return a+b;
}
So - we don't cover all issues....
That's why I think that we check correctness of method, but not all possible variant of method executions.

Reply:
Anyway, you need to try to cover all cases in your code, at least by happy path
Here you try to commit untested part of code and it's a wrong way

Reply:
Ok.
But sorry - I don't agree that this code untested - it tested in DBStorageTest and FileStorageTest.
Also idea shows that all methods are covered with test.
But ok - I'll add tests for this variants of usage.